nx-X11/programs/Xserver/hw/nxagent/Client.h

```diff
@@ -59,11 +59,15 @@ extern int nxagentClientPrivateIndex;
 
 void nxagentInitClientPrivates(ClientPtr);
 
+#undef COUNT_CLIENT_BYTES
+
+#ifdef COUNT_CLIENT_BYTES
 #define nxagentClientAddBytes(pClient, size)	\
   (nxagentClientPriv(pClient) -> clientBytes += (size))
 
 #define nxagentClientBytes(pClient)	\
     (nxagentClientPriv(pClient) -> clientBytes)
+#endif
```

Comment:
COUNT_CLIENT_BYTES will never be defined here? What's the purpose of this change?